include/alp/reference/blas2.hpp

```diff
@@ -29,6 +29,7 @@
 #include <alp/config.hpp>
 #include <alp/rc.hpp>
 #include <alp/matrix.hpp>
+#include <alp/blas3.hpp>
```

Comment:
I guess there is not simple solution, but _blas2.hpp_ should not depend on _blas3.hpp_